docs/cli/token.md

```diff
@@ -74,7 +74,7 @@ The agent token is written to `/var/lib/rancher/k3s/server/agent-token`, in secu
 Support for the `k3s token` command and the ability to join nodes with bootstrap tokens is available starting with the 2023-02 releases (v1.26.2+k3s1, v1.25.7+k3s1, v1.24.11+k3s1, v1.23.17+k3s1).
 :::
 
-K3s supports dynamically generated, automatically expiring agent bootstrap tokens. Bootstrap tokens can only be used to join agents.
+K3s supports dynamically generated, automatically expiring agent [bootstrap tokens](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/). Bootstrap tokens can only be used to join agents and are consumed like the agent tokens with the config `--agent-token` or `K3S_AGENT_TOKEN`.
```

Comment:
```suggestion
K3s supports dynamically generated, automatically expiring agent [bootstrap tokens](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/). Bootstrap tokens can only be used to join agents and are consumed like the agent tokens with the config `--agent-token` or `K3S_AGENT_TOKEN`.
```

This is not correct. You do not need to set --agent-token to use bootstrap tokens. They are created using the CLI  or kubectl.

Reply:
I wanted to express that, to consume the already created bootstrap token, you must use `--agent-token` or `K3S_AGENT_TOKEN`, or is there any other way? We were a bit confused by the matrix https://docs.k3s.io/cli/token#token-types as it was not clear how to consume the bootstrap tokens.

Reply:
On agents, you just pass the token via --token, corresponding config key, or the K3S_TOKEN env var.

--agent-token is set on servers to generate a token that can only be used to join agents. Before bootstrap token support, this was the only way to get an unprivileged token that could not be used to join servers.